pegasus/sites.v3/code.org/views/global/fa/carousel_farsi_how_ai_works_videos.haml

```diff
@@ -0,0 +1,22 @@
+// Should be 9 when done
```

Comment:
I left some of the stuff for the full 9 videos because Jorge is just finishing those up and I think having them around to turn on when ready will be helpful